presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -34,51 +30,28 @@
 import static com.facebook.presto.spi.session.PropertyMetadata.doubleProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.integerProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.stringProperty;
-import static java.lang.String.format;
-import static java.util.Locale.ENGLISH;
 
 public final class IcebergSessionProperties
 {
     private static final String COMPRESSION_CODEC = "compression_codec";
-    private static final String PARQUET_MAX_READ_BLOCK_SIZE = "parquet_max_read_block_size";
     private static final String PARQUET_WRITER_BLOCK_SIZE = "parquet_writer_block_size";
     private static final String PARQUET_WRITER_PAGE_SIZE = "parquet_writer_page_size";
     private static final String PARQUET_WRITER_VERSION = "parquet_writer_version";
-    private static final String PARQUET_USE_COLUMN_NAMES = "parquet_use_column_names";
-    private static final String PARQUET_BATCH_READ_OPTIMIZATION_ENABLED = "parquet_batch_read_optimization_enabled";
-    private static final String PARQUET_BATCH_READER_VERIFICATION_ENABLED = "parquet_batch_reader_verification_enabled";
-    private static final String ORC_BLOOM_FILTERS_ENABLED = "orc_bloom_filters_enabled";
-    private static final String ORC_MAX_MERGE_DISTANCE = "orc_max_merge_distance";
-    private static final String ORC_MAX_BUFFER_SIZE = "orc_max_buffer_size";
-    private static final String ORC_STREAM_BUFFER_SIZE = "orc_stream_buffer_size";
-    private static final String ORC_TINY_STRIPE_THRESHOLD = "orc_tiny_stripe_threshold";
-    private static final String ORC_MAX_READ_BLOCK_SIZE = "orc_max_read_block_size";
-    private static final String ORC_LAZY_READ_SMALL_RANGES = "orc_lazy_read_small_ranges";
-    private static final String ORC_ZSTD_JNI_DECOMPRESSION_ENABLED = "orc_zstd_jni_decompression_enabled";
     private static final String ORC_STRING_STATISTICS_LIMIT = "orc_string_statistics_limit";
-    private static final String ORC_OPTIMIZED_WRITER_ENABLED = "orc_optimized_writer_enabled";
-    private static final String ORC_OPTIMIZED_WRITER_VALIDATE = "orc_optimized_writer_validate";
-    private static final String ORC_OPTIMIZED_WRITER_VALIDATE_PERCENTAGE = "orc_optimized_writer_validate_percentage";
-    private static final String ORC_OPTIMIZED_WRITER_VALIDATE_MODE = "orc_optimized_writer_validate_mode";
     private static final String ORC_OPTIMIZED_WRITER_MIN_STRIPE_SIZE = "orc_optimized_writer_min_stripe_size";
     private static final String ORC_OPTIMIZED_WRITER_MAX_STRIPE_SIZE = "orc_optimized_writer_max_stripe_size";
     private static final String ORC_OPTIMIZED_WRITER_MAX_STRIPE_ROWS = "orc_optimized_writer_max_stripe_rows";
     private static final String ORC_OPTIMIZED_WRITER_MAX_DICTIONARY_MEMORY = "orc_optimized_writer_max_dictionary_memory";
-    private static final String ORC_COMPRESSION_CODEC = "orc_compression_codec";
     private static final String CACHE_ENABLED = "cache_enabled";
```

Comment:
We can remove the getter for this from here as well since it is not used

Reply:
Added comment